shared/native/notification-listeners.desktop.js

```diff
@@ -55,5 +56,8 @@ export default function (dispatch: Dispatch, getState: () => Object, notify: any
     'keybase.1.NotifyPGP.pgpKeyInSecretStoreFile': () => {
       dispatch({type: pgpKeyInSecretStoreFile, payload: undefined})
     },
+    'keybase.1.NotifyUsers.userChanged': ({uid}) => {
```

Comment:
there is a todo here to move this stuff into individual actions , which i agree with. ticketed as https://keybase.atlassian.net/browse/DESKTOP-1851